CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+# 1.85.2
+Bugfix - Disable Digest Resolution for Manifest V2 Schema 1 [#4249](https://github.com/aws/amazon-ecs-agent/pull/4249)
```

Comment:
The rich diff is showing a paragraph of text because each line is missing the `*` that makes it a list item.